Samples/AudioPreprocessing/AudioPreprocessing/Model/PreprocessModel.cs

```diff
@@ -1,16 +1,323 @@
-﻿using System;
```

Comment:
I believe this file / folder is for WPF and not needed. Can we remove?

Reply:
Yes, sorry about the time spent on that folder! I forgot I had left it there...